src/Microsoft.Data.Analysis/DataFrame.cs

```diff
@@ -42,19 +42,36 @@ public DataFrame(IList<DataFrameColumn> columns)
             _columnCollection = new DataFrameColumnCollection(columns, OnColumnsChanged);
         }
 
+        /// <summary>
+        /// Returns the number of rows in this <seealso cref="DataFrame"/>
+        /// </summary>
         public long RowCount => _columnCollection.RowCount;
 
+        /// <summary>
+        /// Returns the columns contained in the <seealso cref="DataFrame"/> as a <seealso cref="DataFrameColumnCollection"/>
+        /// </summary>
         public DataFrameColumnCollection Columns => _columnCollection;
 
         internal IReadOnlyList<string> GetColumnNames() => _columnCollection.GetColumnNames();
 
         #region Operators
+        /// <summary>
+        /// Indexer to get/set values
+        /// </summary>
+        /// <param name="rowIndex"></param>
+        /// <param name="columnIndex"></param>
+        /// <returns></returns>
```

Comment:
Left empty intentionally?

Reply:
No. I should delete the returns tag here. The indexer returns different things based on if its a set or a get method.

Reply:
+1. Please fill out the two params and the returns.